supertokens_python/recipe/session/session_class.py

```diff
@@ -33,21 +32,14 @@
     SessionClaim,
     SessionClaimValidator,
     SessionContainer,
+    GetSessionTokensDangerouslyDict,
 )
 from .constants import protected_props
 from ...framework import BaseRequest
 
 _T = TypeVar("_T")
 
 
-class GetSessionTokensDangerouslyDict(TypedDict):
-    accessToken: str
-    accessAndFrontTokenUpdated: bool
-    refreshToken: Optional[str]
-    frontToken: str
-    antiCsrfToken: Optional[str]
-
-
```

Comment:
Need to discuss about the issue of why the linter does not complain if we comment out `if self.req_res_info is not None:`

Reply:
Surprisingly doing this fixed it. And pyright wasn't complaining about it anyways. Maybe VSCode's linter is outdated/buggy.

```
class SessionContainer(ABC):
    self.req_res_info: Optional[ReqResInfo] = req_res_info
```